### ISSUE TITLE
`dxvk-nvapi/dxvk-versions.json` missing on Ubuntu 22.04

### ISSUE THREAD

**Issue by melianmiko** [lutris/lutris#4350]
Lutris 0.5.10.1~ubuntu22.04.1 (official PPA).

When starting a game, got error becouse `~/.local/share/lutris/runtime/dxvk-nvapi/dxvk-versions.json` don't exist. I don't know why Lutris need this file on AMD Vega 8 GPU, but this can be fixed via single command:

```bash
cp ~/.local/share/lutris/runtime/dxvk/dxvk_versions.json ~/.local/share/lutris/runtime/dxvk-nvapi/
```

Also tried Flatpak version, they works fine. This error appears only with DEB-version.

**Comment by danieljohnson2** [comment]
I think this is a duplicate of #4241, which has a fix merged in master, but just after 0.5.10.1 went.

Those .json files should not be the same actually. Could you delete your dxvk-nvapi/versions.json, and then try the master branch?

You need not install it; just clone the repo and run ./bin/lutris inside the repo. It should download the correct file when when you play a game that needs it. Once you have the right file, the release Lutris should be able to use it too.

**Comment by melianmiko** [comment]
Yep, in `master` branch this bug was fixes.
Thank you, i'll wait for an update.
